chapter_linear-networks/fashion-mnist_vn.md

```diff
@@ -293,11 +300,10 @@ với dấu `@` ở đầu. Ví dụ: @aivivn.
 *
 
 <!-- Phần 2 -->
-*
+* Nguyễn Lê Quang Nhật
```

Comment:
```suggestion
* Nguyễn Lê Quang Nhật
* @rootonchair 
* @aivivn 
* @duythanhvn 
```

Reply:
Dùng tên thật thôi ông giáo ơi, có tên hết rồi nờ.

Ps: tên của aivivn là "trùm cuối"....

Reply:
> Dùng tên thật thôi ông giáo ơi, có tên hết rồi nờ.
> 
> Ps: tên của aivivn là "trùm cuối"....

Có nên để là the  FINAL BOSS ko nhỉ :)))